### PR TITLE
Update Playbook page layout

### LINKED ISSUE #994: Create playbook landing page
The playbook landing page should be created in preparation of the launch of the ebook.

## Tasks
- [x] Implement layout with temporary copy and images
- [x] finalize text
- [x] finalize images

## Resources
- Abstract: https://app.abstract.com/projects/f6401950-547a-11e9-a66d-7f6f3414ea33/branches/1afa758e-8f25-4c17-a533-a71da64f6f36/commits/9a6934239623e0ff0f04d08f719e8e2fa6fea012/files/270f3779-b0f4-43d5-b3ac-9665af17512f/layers/605FBBAE-58FD-4A12-98B7-F0F8C74E5C10?collectionId=feff5db0-94ee-4236-ada1-3483c9749c27&collectionLayerId=68355f0d-fdfa-4d36-93f4-7e39f3fde339

### PR DESCRIPTION
**Pull request by locks**
This PR contains the new layout with the placeholder content. I extracted the CSS for the logos list to the list block in order to re-use it.
I did not extract it to a component since there's only two uses and the design features different clients on both pages.
I also added a new component for the Trainline quote.

### TODO

- [x] finalize wording
- [x] finalize book image
- [x] finalize illustrations
- [x] fix anchor links in Safari
- [x] update the playbook teaser shape and links to use the new illustrations as well (we're leaving the `<ShapeAcrossPlaybook>` as it is since that is only used on the work page anyway and will be replaced by the shape refactoring anyway)
- [x] update photo of book once we have the final illustration for the cover
- [x] remove splitting of the playbook bundle

Closes #994.

### CODE REVIEW

**Comment by ghislaineguerin** [comment]
@locks we also need the negative margin on the bottom shape
image

File: src/ui/components/PagePlaybook/stylesheet.css [hunk 1]
```diff
@@ -8,6 +9,12 @@
   composes: 'layout';
 }
 
+.accent {
```
Comment (marcoow) [inline src/ui/components/PagePlaybook/stylesheet.css:15]: This does not seem to be used anywhere?

File: src/ui/components/ShapePlaybook/component-test.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import hbs from '@glimmer/inline-precompile';
+import { render } from '@glimmer/test-helpers';
+import { setupRenderingTest } from '../../../utils/test-helpers/setup-rendering-test';
+
+const { module, test } = QUnit;
+
+module('Component: ShapeFeature', function(hooks) {
```
Comment (marcoow) [inline src/ui/components/ShapePlaybook/component-test.ts]:
```suggestion
module('Component: ShapePlaybook', function(hooks) {
```

File: src/ui/components/ShapePlaybook/component-test.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    await render(hbs`<ShapeFeature />`);
```
Comment (marcoow) [inline src/ui/components/ShapePlaybook/component-test.ts]:
```suggestion
    await render(hbs`<ShapePlaybook />`);
```

File: src/ui/components/ShapePlaybook/component-test.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    assert.ok(this.containerElement.querySelector('div'));
+  });
+
+  module('when invoked with a @srcset', function() {
```
Comment (marcoow) [inline src/ui/components/ShapePlaybook/component-test.ts]: This should not be needed

File: src/ui/components/ShapePlaybook/component-test.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    test('it renders figure and figcaption elements', async function(assert) {
+      await render(hbs`<ShapeFeature @srcset="./some/image/source.png" />`);
+
+      assert.ok(this.containerElement.querySelector('figure'));
+      assert.ok(this.containerElement.querySelector('figcaption'));
+    });
+  });
+
+  module('when invoked without a @srcset', function() {
```
Comment (marcoow) [inline src/ui/components/ShapePlaybook/component-test.ts]: This should not be needed

File: src/ui/components/ShapeQuoteTrainline/component-test.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import hbs from '@glimmer/inline-precompile';
+import { render } from '@glimmer/test-helpers';
+import { setupRenderingTest } from '../../../utils/test-helpers/setup-rendering-test';
+
+const { module, test } = QUnit;
+
+module('Component: ShapeQuoteCardstack', function(hooks) {
```
Comment (marcoow) [inline src/ui/components/ShapeQuoteTrainline/component-test.ts]:
```suggestion
module('Component: ShapeQuoteTrainline', function(hooks) {
```

File: src/ui/components/ShapeQuoteTrainline/component-test.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    await render(hbs`<ShapeQuoteCardstack />`);
```
Comment (marcoow) [inline src/ui/components/ShapeQuoteTrainline/component-test.ts]:
```suggestion
    await render(hbs`<ShapeQuoteTrainline />`);
```

File: src/ui/components/ShapeQuoteTrainline/stylesheet.css [hunk 8]
```diff
@@ -0,0 +1,7 @@
+@block typography from "../../styles/blocks/typography.block.css";
+
+:scope {
+  block-name: ShapeQuoteCardstack;
```
Comment (marcoow) [inline src/ui/components/ShapeQuoteTrainline/stylesheet.css]:
```suggestion
  block-name: ShapeQuoteTrainline;
```

File: src/ui/components/ShapeQuoteTrainline/template.hbs [hunk 9]
```diff
@@ -0,0 +1,14 @@
+<ShapeFeature
+  @srcset="/assets/images/work/trainline@600.jpg 600w, /assets/images/work/trainline@1200.jpg 1200w"
+  @sizes="(max-width: 887px) 600px, 1200px"
+  @srcAlt="Cardstack"
```
Comment (marcoow) [inline src/ui/components/ShapeQuoteTrainline/template.hbs]:
```suggestion
  @srcAlt="Trainline"
```

**Review by marcoow — CHANGES_REQUESTED** [review]
This looks like in Abstract of course but I think it's a bit strange since _"Organization of this book"_ is actually closer to the previous section than the one it actually belongs to…

Bildschirmfoto 2020-03-20 um 14 41 37

File: src/ui/components/PagePlaybook/stylesheet.css (hunk 1 above)
Comment (locks) [reply src/ui/components/PagePlaybook/stylesheet.css:15]: It is used in the "Organization of this book" numbering:

```
      <div block:class="accent">
        01
      </div>
```

Comment (marcoow) [reply src/ui/components/PagePlaybook/stylesheet.css:15]: Ah, missed that. We should not set font sizes in `px` thought but be consistent in using `rem`.

**Comment by locks** [comment]
@ghislaineguerin do you have mobile viewport designs for the playbook download shape?

image

**Comment by locks** [comment]
> @locks we also need the negative margin on the bottom shape
Apologies @ghislaineguerin, I do not understand which shape this applies to.

**Comment by ghislaineguerin** [comment]
@locks 
here's the mobile version
https://share.goabstract.com/d8bc6dbd-34f7-4249-a655-342aa6758535?mode=design&sha=fbe8b5e3307cb0f350275d304dd046e389cfb16a

**Comment by marcoow** [comment]
@ghislaineguerin please review but don't merge yet

**Review by marcoow — APPROVED** [review]

**Review by ghislaineguerin — APPROVED** [review]
looks good

**Pull request merged**